schedule/yast/minimal+base/minimal+base@yast-svirt-hyperv.yaml

```diff
@@ -31,6 +31,7 @@ schedule:
   - console/zypper_ref
   - console/ncurses
   - console/glibc_sanity
+  - console/yast2_bootloader
```

Comment:
As mentioned before, let's not use minimal+base and extend yast2_ncurses textmode with virtualization. Same for s390x

Reply:
Sorry but cant see any good reason for this atm. it does not make much difference and minimal+base is a good candidate

Reply:
ok, so you still want to spread those yast modules in another test suite instead of extending the one that we have for yast modules(extending => (1) apply conditional schedule on the existing file or (2) creating another yaml files but with similar test suite name). The only difference is a little of organization but maybe I nit-picking, @rwx788 wdyt?

Reply:
Can we merge this?

Reply:
At least two of us thought that spreading was not a good idea, I'd like to hear from @rwx788 to collect more opinions before merging.

Reply:
So I'm also against mixing scenarios this way. We have now 3-4 test suites for testing yast modules and I would reuse those. In case you believe that minimal setup is good environment for testing it, we can chain them and boot into the image this job creates.

Reply:
honestly i cant see any reason for this. In addition, the PR is open for some days and any change will require a new iteration of VR, reviews and so on.  I mentioned that in a previous comment but the minimal+base is also good candidate for this module. The requirements were to cover the arch/backends and i did not foresee any reason to run more jobs for something like that. With the same explanation i would not disagree with the minimal setup as now, just for this. In the minimal setup i presume that it would be less possible to catch some defects

Reply:
So minimal+base removed.